docker-compose.yml

```diff
@@ -13,6 +13,8 @@ crossdock:
 
       - BEHAVIOR_DANCE=client,server,transport
       - BEHAVIOR_SING=client,server,transport
+      - BEHAVIOR_RUN=client,server,transport
+      - BEHAVIOR_SKIP_RUN=client=alpha+server=alpha
```

Comment:
Perhaps we should call this `FILTER_` or `PRUNE_` to distinguish from tests skipped for other reasons.